test/fixtures/test-runner/output/retryable.js

```diff
@@ -0,0 +1,49 @@
+const common = require('../../../common');
+const test = require('node:test');
+const assert = require('node:assert');
+
+test.describe('hardcoded retries', () => {
+  test('should retry 3 times (success)', {
+    retries: 5,
+  }, common.mustCallAtLeast((t) => {
+    if (t.currentAttempt < 3) {
+      throw new Error('fail');
+    }
+  }, 3));
+
+  // Expect: Failure
+  test('should retry 5 times (failure)', {
+    retries: 5,
+  }, common.mustCallAtLeast(() => {
+    throw new Error('fail');
+  }, 5));
+});
+
+test.describe('dynamic retries', () => {
+  test('should retry 3 times (success)', common.mustCallAtLeast((t) => {
+    t.retries(5);
+    if (t.currentAttempt < 3) {
+      throw new Error('fail');
+    }
+  }, 3));
+
+  test('should retry 3 times (failure)', common.mustCallAtLeast((t) => {
+    t.retries(3);
+    throw new Error('fail');
+  }, 3));
```

Comment:
I think this needs a test for calling `t.retries()` multiple times (should it fail, what is the expected behavior)

Reply:
It won't fail, it will just update the number of retries. I'll fix that tomorrow, thank you!